<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIAB</class>
 <widget class="QMainWindow" name="CIAB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>214</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CIAB</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 1 #3AAFA9 ,
	stop: 0 #FEFFFF);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #17252A;</string>
    </property>
    <property name="text">
     <string>Chemistry In a Box</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>50</y>
      <width>251</width>
      <height>308</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>14</number>
     </property>
     <item>
      <widget class="QPushButton" name="periodicTable">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	border: 1px solid black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #E08D8C ,
	stop: 1 #E35F55);
}

QPushButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #995151 ,
	stop: 1 #995151);
}</string>
       </property>
       <property name="text">
        <string>Dynamic Periodic Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="molecularWeight">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	border: 1px solid black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #E08D8C ,
	stop: 1 #E35F55);
}

QPushButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #995151 ,
	stop: 1 #995151);
}</string>
       </property>
       <property name="text">
        <string>Molecular Weight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="elementsBusters">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	border: 1px solid black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #E08D8C ,
	stop: 1 #E35F55);
}

QPushButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #995151 ,
	stop: 1 #995151);
}</string>
       </property>
       <property name="text">
        <string>Elements Busters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dailycompounds">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	border: 1px solid black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #E08D8C ,
	stop: 1 #E35F55);
}

QPushButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #995151 ,
	stop: 1 #995151);
}</string>
       </property>
       <property name="text">
        <string>Daily Compounds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="equationbalancer">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: white;
	border: 1px solid black;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #E08D8C ,
	stop: 1 #E35F55);
}

QPushButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	stop: 0 #995151 ,
	stop: 1 #995151);
}</string>
       </property>
       <property name="text">
        <string>Equation Balancer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
